charts/ratify/templates/secret.yaml

```diff
@@ -1,10 +1,11 @@
+{{- if eq .Values.akvCertConfig.enabled false }}  
 apiVersion: v1
 kind: Secret
 metadata:
   name: {{ include "ratify.fullname" . }}-notary-certificate
 data:
-  ratify-test.crt: {{ .Values.ratifyTestCert | b64enc | quote }}
-
+  notary.crt: {{ required "You must provide .Values.notaryCert" .Values.notaryCert | b64enc | quote }}
```

Comment:
what if users only provide inline cert store? I'm thinking if we can deprecate this secret since we already support inline cert store now. The file system based option was to support old notation version without ability to retrieve certs in memory.

Reply:
great point Binbin, i will explore switching this to inline certstore